typescript/cli/package.json

```diff
@@ -44,6 +45,7 @@
     "lint": "eslint . --ext .ts",
     "prettier": "prettier --write ./src ./examples",
     "test": "mocha --config .mocharc.json",
+    "test:e2e": "./scripts/e2e-test.sh",
```

Comment:
These tests are not enabled in CI yet. Will do this: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/4388